### PR TITLE
*added endpoint handbook

### PR DESCRIPTION
**Pull request by graz-dev**
## Description

Added endpoint handbook to explain creation of each type of endpoint

## Pull Request Type

- [X] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [X] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [X] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [X] No sensible content has been committed

### CODE REVIEW

File: docs/getting-started/handbooks/console/endpoints.mdx [hunk 1]
```diff
@@ -0,0 +1,121 @@
+---
+id: endpoints
+title: Endpoints
+sidebar_label: Endpoints
+---
+
+In this guide, we will see how to configure the various types of endpoints available in my platform console, including:
+- [CRUD](/getting-started/handbooks/console/endpoints.mdx#create-a-crud-endpoint)
```
Comment (akelity) [inline docs/getting-started/handbooks/console/endpoints.mdx:8]:
```suggestion
- [CRUD](#create-a-crud-endpoint)
```
You can only use the anchor for page links.

File: docs/getting-started/handbooks/console/endpoints.mdx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+## Create a CRUD Endpoint
+
+Creating an endpoint of type **CRUD** hooks your endpoint directly to one of your CRUDs collections defined in `Data Models/MongoDB CRUD` section in Mia-Platform Console.
+
+:::caution 
+To create a CRUD Collection it is necessary to have a CRUD Service installed, to do so you simply create a new Microservice from Marketplace choosing **CRUD Service** from the catalog.
+:::
+
+To create a CRUD endpoint you have to move in the `Endpoint` section of the Mia-Platform Console, here you can create new endpoints.
```
Comment (akelity) [inline docs/getting-started/handbooks/console/endpoints.mdx:42]:
```suggestion
To create a CRUD endpoint you have to move in the `Endpoint` section of the Mia-Platform Console, here you can create new endpoints.
```

This sentence is always repeated, let's put it at the beginning with a screenshot of the page (with navigation).

**Review by akelity — COMMENTED** [review]
In addition to removing the repeated part, let's remove the navigation bar from the screenshots of the individual endpoints.

File: docs/getting-started/handbooks/console/endpoints.mdx (hunk 1 above)
Comment (graz-dev) [reply docs/getting-started/handbooks/console/endpoints.mdx:8]: Done

**Review by akelity — APPROVED** [review]

**Comment by akelity** [comment]
LGTM

**Pull request merged**